test/annotate_jl.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# @Author: ZwEin
+# @Date:   2016-10-06 20:17:15
+# @Last Modified by:   ZwEin
+# @Last Modified time: 2016-10-06 21:02:50
+
+import os
+import sys
+import json
+import codecs
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+TEST_DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
+
+from digAgeExtractor.age_helper import *
+
+def annotate(intput_path, output_path=None):
```

Comment:
Hi Jason, sorry for late response, :p. I just removed the file and pushed the change.